packages/modals/src/styled/StyledTooltipModal.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import { Placement } from '@popperjs/core';
+import { arrowStyles, retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { getArrowPosition } from '../utils/gardenPlacements';
+
+const COMPONENT_ID = 'modals.tooltip_modal';
+
+export interface IStyledTooltipModalProps {
+  hasArrow?: boolean;
+  /**
+   * All valid [Popper.JS Placements](https://popper.js.org/popper-documentation.html#Popper.placements)
+   */
+  placement: Placement;
+  isAnimated?: boolean;
+}
+
+/**
+ * Accepts all `<div>` props
+ *
+ * 1. Override arrow parent positioning to ensure arrow is visible beyond block
+ *    overflow boundaries.
+ */
+export const StyledTooltipModal = styled.div.attrs<IStyledTooltipModalProps>(props => ({
+  'data-garden-id': COMPONENT_ID,
+  'data-garden-version': PACKAGE_VERSION,
+  className: props.isAnimated && 'is-animated'
+}))<IStyledTooltipModalProps>`
+  /* stylelint-disable-next-line declaration-no-important */
+  position: static !important; /* [1] */
+
+  padding: ${props => props.theme.space.base * 5}px;
+  width: 400px;
```

Comment:
Hard coding a width here forces consumers to override CSS anything `400px` doesn't work for their context. Currently running into this situation using the tooltip modal in the color picker and color swatch dialogs.

Reply:
I believe the solution is to `!important` as we do with https://github.com/zendeskgarden/react-components/blob/2ad0042eb3bea0ed6a93be02b3b2e2333aff183d/packages/colorpickers/src/styled/ColorpickerDialog/StyledTooltipModal.ts#L23.